utils/bashrc.d/10-pdcli.bashrc

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+#set -eEuo pipefail
+
+# pdlo will login to the pd service
+alias pdlo='pd login'
+
+# pdsr will show resolved incidents for a service
+function pdsr() {
+  local service=$1
+  pd incident:list -s resolved -S $service
+}
+
+SILENT_TEST_USER_ID=P8QS6CC
```

Comment:
Might be a personal preference, but maybe group "global"-type variables at the top of the file for quick reference?

Reply:
yeah, I can do that, no problem :)